app/data/models.py

```diff
@@ -34,7 +35,45 @@ class ActionTypes(object):
             (UPDATE, 'Update'),
             (DELETE, 'Delete')
         )
+
         @classmethod
         def as_list(cls):
             return [cls.CREATE, cls.UPDATE, cls.DELETE]
+
     action = models.CharField(max_length=6, choices=ActionTypes.choices)
+
+
+class CeleryJob(models.Model):
```

Comment:
Yeah, I think this is the only case where we actually really care about when the last task was run since we'll need to make use of that value in the task itself for optimization.